apps/omg_watcher/lib/omg_watcher/exit_processor/tools.ex

```diff
@@ -96,14 +96,15 @@ defmodule OMG.Watcher.ExitProcessor.Tools do
           list(%{
             call_data: map(),
             root_chain_txhash: charlist(),
-            log_index: non_neg_integer()
+            log_index: non_neg_integer(),
+            eth_height: pos_integer()
           })
   def to_bus_events_data(eth_events_with_exiting_utxos) do
     Enum.reduce(eth_events_with_exiting_utxos, [], &to_bus_events_reducer/2)
   end
 
   defp to_bus_events_reducer(
-         {%{root_chain_txhash: root_chain_txhash, log_index: log_index}, utxo_positions},
+         {%{root_chain_txhash: root_chain_txhash, log_index: log_index, eth_height: eth_height}, utxo_positions},
```

Comment:
these functions are pretty hard to digest, maybe we should stop putting map pattern matching in function clauses.